docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -608,9 +608,10 @@ And the following unique queues:
 - `ENABLE_AUTO_REGISTRATION`: **false**: Automatically create user accounts for new oauth2 users.
 - `USERNAME`: **nickname**: The source of the username for new oauth2 accounts:
   - `userid` - use the userid / sub attribute
-  - `nickname` - use the nickname attribute
+  - `nickname` - use the nickname
+  - `preferred_username` - use the preferred_username
   - `email` - use the username part of the email attribute
-  - Note: `nickname` and `email` options will normalize input strings using the following criteria:
+  - Note: `nickname`, `preferred_username` and `email` options will normalize input strings using the following criteria:
```

Comment:
Could it also update the `app.example.ini`?

https://github.com/go-gitea/gitea/blob/c77e8140bc2ac6521dbebfb77613dce2648bfcb8/custom/conf/app.example.ini#L1553-L1561